lsp-ui-doc.el

```diff
@@ -776,11 +752,15 @@ before, or if the new window is the minibuffer."
         (cl-callf copy-tree frameset-filter-alist)
         (push '(lsp-ui-doc-frame . :never) frameset-filter-alist)))
     (add-hook 'post-command-hook 'lsp-ui-doc--make-request nil t)
-    (add-hook 'delete-frame-functions 'lsp-ui-doc--on-delete nil t))
+    (add-hook 'delete-frame-functions 'lsp-ui-doc--on-delete nil t)
+    (remove-hook 'focus-in-hook 'posframe--redirect-posframe-focus t)
+    (setq posframe-mouse-banish nil))
    (t
     (lsp-ui-doc-hide)
     (remove-hook 'post-command-hook 'lsp-ui-doc--make-request t)
-    (remove-hook 'delete-frame-functions 'lsp-ui-doc--on-delete t))))
+    (remove-hook 'delete-frame-functions 'lsp-ui-doc--on-delete t)
+    (add-hook 'focus-in-hook 'posframe--redirect-posframe-focus nil t)
+    (setq posframe-mouse-banish t))))
```

Comment:
I don't like setting it back to t since it may have been nil initially.IMO we should coordinate with upstream what is the desired behaviour.

Reply:
According to tumashu/posframe#54, that option could probably be safely removed from posframe. It seems, according to [documentation](https://github.com/tumashu/posframe#customizing-mouse-pointer-control), that it's intended to be set from init.el, so it can probably be removed from here.